cmd/ocm-backplane/console/console_suite_test.go

```diff
@@ -1,8 +1,8 @@
-package console
+package console_test
```

Comment:
Minor nit, lets set the package naming standard similar to the other tests.
```suggestion
package console
```